generated-platform-project/pom.xml

```diff
@@ -73,7 +73,7 @@
           <plugin>
             <groupId>org.sonatype.plugins</groupId>
             <artifactId>nexus-staging-maven-plugin</artifactId>
-            <version>1.6.13</version>
```

Comment:
@loicmathieu could you please try re-generating the platform to see if it results in a diff? I just tried upgrading the GCS version, `mvn -Dsync` and I didn't get this change. Thanks.

Reply:
Just did it, no change